lumen/ai/agents.py

```diff
@@ -375,6 +340,82 @@ async def respond(
         return table_list
 
 
+class LumenBaseAgent(Agent):
+
+    user = param.String(default="Lumen")
+
+    prompts = param.Dict(
+        default={
+            "retry_output": {
+                "response_model": RetrySpec,
+                "template": PROMPTS_DIR / "LumenBaseAgent" / "retry_output.jinja2"
+            },
+        }
+    )
+
+    _output_type = LumenOutput
+
+    _max_width = None
+
+    def _update_spec(self, memory: _Memory, event: param.parameterized.Event):
+        """
+        Update the specification in memory.
+        """
+
+    def _render_lumen(
+        self,
+        component: Component,
+        message: pn.chat.ChatMessage = None,
+        messages: list | None = None,
+        render_output: bool = False,
+        title: str | None = None,
+        **kwargs
+    ):
+        memory = self._memory
+
+        async def retry_invoke(event: param.parameterized.Event):
+            reason = event.new
+            modified_messages = mutate_user_message(
+                f"New feedback: {reason!r}.\n\nThese were the previous instructions to use as reference:",
+                deepcopy(messages), wrap='\n"""\n', suffix=False
+            )
+            with self.param.update(memory=memory):
+                system = await self._render_prompt(
+                    "retry_output", messages=modified_messages, spec=out.spec,
+                    language=out.language
```

Comment:
Seems like you removed the language from the template again, if that was intentional remove it here too.

Reply:
I think you might not have pushed your changes on language(?) I don't see it in https://github.com/holoviz/lumen/pull/851/commits/da41924e4f24930d6cc71a21be2bedbd45bad5ac